CONTRIBUTING.rst

```diff
@@ -0,0 +1,61 @@
+How to Contribute
+=================
+
+We can use contributions to the images here, but can also use your help in
+testing the images. If you'd like to sign up for beta testing the ``latest``
+image, sign up here:
+
+https://goo.gl/forms/3oAbxkGMUiEZz2782
+
+We'll need to verify that you are the account holder, and that you made the
+request for the projects that you listed.
```

Comment:
Not sure to understand this. Maybe because my english sucks or maybe because it's not clear enough :)

The idea is that people sign up there and one person from the Operations goes to the production environment and change the settings for that specific project to start using the `latest` image on production but only on those projects that the person has mentioned. Am I right?

> but can also use your help in testing the images

That is the part that makes me feel confused. Because it says "testing the images" but then I have to suscribe with that form but the following section explains me about how to test it in my own computer.

If the _testing_ will be done automatically in production environment I think we need to clarify that and also _promote it_ in a better way like: "Hey you can help us by trying the latest changes in the RTD builder option. It's pretty easy, just fill this form with your project's information and start opening issues if something goes wrong!" :grin: , or something like that, hehe